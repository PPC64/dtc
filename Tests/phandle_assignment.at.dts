/dts-v1/;

/{
   quxnode {
      qux = <&BAZ>;
   };
   quuxnode {
      quux = <1 2 3 4 &FOO>;
   };

   BAZ: baz { };
   FOO: foo {
      BAR: bar { };
      foobar {
         FOOBARCHILD: foobarchild { };
      };
   };
};
