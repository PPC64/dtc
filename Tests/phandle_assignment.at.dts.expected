/dts-v1/;

/  {

	quxnode {

		qux = <0x1>;
	};
	quuxnode {

		quux = <0x1 0x2 0x3 0x4 0x2>;
	};
	baz {

		phandle = <0x1>;
	};
	foo {

		phandle = <0x2>;
		bar {

			phandle = <0x3>;
		};
		foobar {

			foobarchild {

				phandle = <0x4>;
			};
		};
	};
	__symbols__ {

		FOOBARCHILD = "/foo/foobar/foobarchild";
		BAR = "/foo/bar";
		FOO = "/foo";
		BAZ = "/baz";
	};
};
